sycl/test/regression/2020-spec-constants-debug-info.cpp

```diff
@@ -0,0 +1,27 @@
+// RUN: %clangxx -fsycl -g %s -o %t.out
+// 
+// The idea of this test is to make sure that we can compile the following
+// simple example without crashes/assertions firing at llvm-spirv step due to
+// debug info corrupted by sycl-post-link
```

Comment:
This looks like an overkill to me.
Please, write a unittest for `sycl-post-link` tool and move such test to llvm-test-suite.

Reply:
Removed the test in e49fb2260e9b672a53779901e006cac04ecd82ab, outlined it into intel/llvm-test-suite#307, updated LIT tests for `sycl-post-link` in 81fc86f6e25c3ea82538b68f73f7c08ed605f042